scripts/install/jdownloader.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+# JDownloader Installer for swizzin
+# Author: Aethaeran
+
+# References
+# https://swizzin.ltd/dev/structure/
+# https://www.digitalocean.com/community/tutorials/how-to-install-java-with-apt-on-debian-10
+# https://support.jdownloader.org/Knowledgebase/Article/View/install-jdownloader-on-nas-and-embedded-devices
+# Liara already made a doc for installing JDownloader manually https://docs.swizzin.net/guides/jdownloader/
+# https://linuxize.com/post/how-to-check-if-string-contains-substring-in-bash/
+# https://linuxize.com/post/bash-check-if-file-exists/
+
+# TODO: Figure out if these imports are even necessary
+. /etc/swizzin/sources/globals.sh
+. /etc/swizzin/sources/functions/users
+
+# Functions
+function install_jdownloader() {
+
+    # Get my.jdownloader info for user
+    # TODO: Should double check to confirm everything is accurate, and loop back if anything isn't filled out.
+    # TODO: swizzin likely has utils for this already
+    echo_query "The user will require an account at https://my.jdownloader.org/ in order to access your JDownloader installation's web UI.\nIt is recommended to use a randomly generated password for the account since the password is save in plain text in the program folder.\nEnter the e-mail used to access this account once one is created:"
```

Comment:
I'd break this up a bit
```suggestion
    echo_info "The user will require an account at https://my.jdownloader.org/ in order to access your JDownloader installation's web UI.\nIt is recommended to use a randomly generated password for the account since the password is save in plain text in the program folder."
    echo_query "Enter the e-mail used to access this account once one is created:"
```

Reply:
Also a good call. Honestly still haven't looked at the differences between the different swizzin echos. But this another time where I get a bit wordy. I'm sure this could be condensed further still.